lib/shoryuken/manager.rb

```diff
@@ -78,6 +78,7 @@ def processor_done(queue, processor)
     def processor_died(processor, reason)
       watchdog("Manager#processor_died died") do
         logger.error { "Process died, reason: #{reason}" unless reason.to_s.empty? }
+        reason.backtrace.each { |bt| logger.error { bt } } unless reason.backtrace.nil?
```

Comment:
@barisbalic could you give a sample output showing how it looks like? I'm wondering if the backtrace goes all the way to the worker that died or if it will always shows the same backtrace listing something internal only.

WDYT?

```ruby
logger.error { reason.backtrace.to_a.join("\n") }
```

But be careful, I remember getting a lot of `reason = nil` when I was testing `processor_died`. So we may need `if reason and !reason.backtrace.to_a.empty?` cc/ @mariokostelac